lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -365,7 +368,7 @@ func (c *Client) startInputStreaming(stopCh chan struct{}) error {
 	defer c.cfg.Logger.InfoContext(context.Background(), "TDP input streaming finished")
 
 	// Remember mouse coordinates to send them with all CGOPointer events.
-	var mouseX, mouseY uint32
+	var withheldResize *tdp.ClientScreenSpec
```

Comment:
The comment about remembering mouse coordinates no longer seems accurate here.

Reply:
f36db5e5f1c7dcaa346b1438557a8b6005c5ba35